include/command_graph_generator.h

```diff
@@ -105,15 +106,108 @@ class command_graph_generator {
 
 	command_graph& get_command_graph() { return m_cdag; }
 
+	/// Only for testing: Instead of (at most) a single chunk per node, generate `multiplier` chunks per node for each task.
+	/// This is to ensure that CDAG generation logic works for arbitrary numbers of chunks, even though we don't provide
+	/// a way for users to specify more than one chunk per node... yet.
+	void set_test_chunk_multiplier(const size_t multiplier) {
```

Comment:
```suggestion
	void test_set_chunk_multiplier(const size_t multiplier) {
```
So that "intrusive" test functions have the `test_` prefix like they do everywhere else.

Reply:
If by "everywhere" you mean runtime.h - ok, done!